README.md

```diff
@@ -292,6 +292,34 @@ Dies erzeugt folgende Ausgabe:
 <meta name="twitter:image:alt" content="Der Bildtitel aus dem Medienpool" />
 ```
 
+## Meta-Tags erweitern / ändern
+
+```php
+\rex_extension::register('YREWRITE_SEO_TAGS', function(\rex_extension_point $ep) {
```

Comment:
```suggestion
rex_extension::register('YREWRITE_SEO_TAGS', function(rex_extension_point $ep) {
```

Ich würde in der Doku beim einheitlichen Schema bleiben. Und da REDAXO bisher selbst keine Namespaces nutzt, auch hier ohne Backslashes.